plugins/wallabag.koplugin/main.lua

```diff
@@ -32,84 +41,83 @@ local util = require("util")
 local _ = require("gettext")
 local T = FFIUtil.template
 
--- constants
+-- Constants
 local article_id_prefix = "[w-id_"
 local article_id_postfix = "] "
 local failed, skipped, downloaded = 1, 2, 3
 
-local Wallabag = WidgetContainer:extend{
+local Wallabag = WidgetContainer:extend {
```

Comment:
I appreciate your enthusiasm, but commit https://github.com/koreader/koreader/pull/12949/commits/65d86be1349e522c18bb1c2a73b340c4e76366e4 has a very high "please don't" factor. :-)

Anything that changes something like `extend{}` to `extend {}` goes not only against our particular code style dialect, but against basically all Lua code to boot.

Reply:
Sorry, I figured the Lua LSP plus the provided `.editorconfig` would follow your code style, but I take it, it doesn't. I'll change this back